### DISCUSSION TITLE
Is there a examples of usage with Rspack?

### DISCUSSION THREAD

**Discussion by krutoo** [mdx-js/mdx#2490] [Q&A] [answered]
Hi, im trying to use `@mdx-js/loader` in Rspack but looks like im doing something wrong because i have error `Module parse failed...`

Can someone provide link to example repo/codeandbox of usage `@mdx-js/loader` wit Rspack?

**Comment by wooorm** [comment, marked as answer]
rspack is not maintained here -- they have a website though, and they have their own MDX integration, which I’m sure is documented on their website.

I made https://github.com/wooorm/mdxjs-rs, so I’d recommend that.

**Reply by ChristianMurphy** [reply]
Adding on it sounds like @krutoo is looking for https://rspress.dev/
Which is by the same maintainer as rspack and uses the rust mdx implementation by default.

**Reply by krutoo** [reply]
> Adding on it sounds like @krutoo is looking for https://rspress.dev/
> 
> Which is by the same maintainer as rspack and uses the rust mdx implementation by default.

@ChristianMurphy im in search of example of using mdx exactly in rspack, not rspress or rsbuild

**Reply by ChristianMurphy** [reply]
Okay 🤷 
This feels a lot like an XY problem, if you want a static site, use their static site builder.

But to address your more technical concern.

Rspack claims to be webpack compatible.
If the web pack example doesn't work, it's likely an rspack bug.
They've found and resolved several already https://github.com/web-infra-dev/rspack/issues?q=is%3Aissue+mdx

Check if your issue matches one of the existing tickets.
And if it doesn't log a new one with enough details they can reproduce and fix the issue.
